### PR TITLE
meetings filter by member name

### PR DESCRIPTION
**Pull request by zasyadev**

### CODE REVIEW

File: component/Meetings/MeetingsList.jsx [hunk 1]
```diff
@@ -17,19 +18,22 @@ import { openNotificationBox } from "../common/notification";
...
+  const { membersList } = MemberListHook(user);
 
   async function fetchMeetingList() {
     setLoading(true);
     setMeetingsList([]);
 
     await httpService
-      .get(`/api/meetings`)
+      .get(`/api/meetings?filterId=${filterId}`)
       .then(({ data: response }) => {
```
Comment (aniljailta) [inline component/Meetings/MeetingsList.jsx:37]: better to ignore .then (). when inside a asyc function. retreive value directly

File: component/Review/ReviewAssignessModal.jsx [hunk 2]
```diff
@@ -101,7 +101,7 @@ const GetAssigneesList = ({ type, reviewAssignee }) => {
             </div>
 
             <div className="flex-1">
-              <p className="mb-0 text-primary font-medium text-base">
+              <p className="mb-0  font-medium text-base">
                 {assignee?.assigned_to?.first_name}
```
Comment (aniljailta) [inline component/Review/ReviewAssignessModal.jsx:105]: when sure  that there will always be assignee dont use .? operator

File: component/Meetings/MeetingsList.jsx (hunk 1 above)
Comment (zasyadev) [reply component/Meetings/MeetingsList.jsx:37]: okay will try this from now on

File: component/Review/ReviewAssignessModal.jsx (hunk 2 above)
Comment (zasyadev) [reply component/Review/ReviewAssignessModal.jsx:105]: okay

**Pull request merged**